filters/awssigner/awssigv4/doc.go

```diff
@@ -0,0 +1,52 @@
+/*
+
+Package sigv4 signs requests using aws signature version 4 mechanism. see https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html
+
+# Filter awsSigv4
+
+Example usage -
+	`editorRoute: * -> awsSigv4("dynamodb" , "us-east-1", false, false, false) -> "https://dynamodb.us-east-1.amazonaws.com";`
+
+	This filter expects
```

Comment:
I think you should also have the definition of the filter, such that the user can relate to the docs below and understand the exa,ple above:

```
awsSigv4("<service>, "<region>", <DisableHeaderHoisting>, <DisableURIPathEscaping>, <DisableSessionToken>)
```

Reply:
Added definition in both `doc.go` and `filter.md`, Thanks